src/Dropdown/Dropdown.js

```diff
@@ -19,23 +19,60 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   constructor() {
     super();
     this.state = {
+      menuDirection: 'down',
+      menuHeight: null,
       isOpen: false,
-      selectedID: null
+      selectedID: null,
+      knowMenuHeight: false
     };
   }
 
+  componentDidUpdate() {
+    // If we don't know the menu height already, we need to calculate it after
+    // it's rendered. It's rendered inside a concealed container, so it's okay
+    // if it renders in the wrong direction.
+    if (!this.state.knowMenuHeight) {
+      let dropdownMenuConcealer = React.findDOMNode(
+        this.refs.dropdownMenuConcealer
+      );
+      let menuDirection = this.state.menuDirection;
+      let menuHeight = this.state.menuHeight;
+
+      if (dropdownMenuConcealer != null) {
+        // Get the height and direction of the concealed menu.
+        menuHeight = dropdownMenuConcealer.children[0].clientHeight;
```

Comment:
Could we find the `child` in a different way? 
You're accessing reacts internal API. If we can do this with like `querySelector` that would be nicer.